src/ui/buttons/props.js

```diff
@@ -762,18 +761,22 @@ export function normalizeButtonMessage(
 
   if (typeof offer !== "undefined") {
     if (!Array.isArray(offer)) {
-      throw new TypeError(
-        `Expected message.offer to be an array of strings, got: ${String(
-          offer
-        )}`
-      );
+      if (typeof offer !== "string") {
+        throw new TypeError(
+          `Expected message.offer to be an array of strings, got: ${String(
+            offer
+          )}`
+        );
+      }
+      offer = [offer];
```

Comment:
If the input to this function on its second call is a string of multiple offers (e.g. `"pay_later_long_term,pay_later_short_term"`), is this then going to result in one large single string entry `["pay_later_long_term,pay_later_short_term"]` which will then cause the invalid offer error to be thrown?

Reply:
It will throw an invalid offer error before it even run the offer check as the offer input looks for `$ReadOnlyArray<$Values<typeof MESSAGE_OFFER>>` which can only be `pay_later_long_term` or `pay_later_short_term`, no multi offer. If we did want multiple offer, then we will have to add those cases to the MESSAGE_OFFER enum.

Reply:
`$ReadOnlyArray<$Values<typeof MESSAGE_OFFER>>` is inclusive of an input of `["pay_later_short_term", "pay_later_long_term"]`. If I pass that in I get the following error:
```
Uncaught Error: Error: Invalid offer(s): pay_later_short_term,pay_later_long_term
```

Reply:
I just realized the actual error and it has been fixed!!!